<?xml version="1.0"?>
<launch>
  <node name="uav_node" pkg="serial_ros" type="comm_read_write" output="screen" >
        <param name="serial_port_name" value="/dev/ttyUSB0" />
        <!-- <param name="baudrate" value="B9600" /> -->
        <param name="Hz" value="30.0" />
        <param name="is_uav_terminal" value="true" />
        <remap from="cmd_vel" to="/uav1/cmd_vel"/>
        <remap from="truth_state" to="/uav1/ground_truth/state"/>
  </node>
</launch>
